### PR TITLE
Include jinja2 templates in the build artifact

### PR DESCRIPTION
**Pull request by marcospri**
Previous commit removed MANIFEST which included templates as part of the
build.

Using a different mechanism here that can be specified directly in
setup.cfg

https://setuptools.pypa.io/en/latest/userguide/datafiles.html#package-data

# Testing 

- On checkmate's repo checkout the branch `dependabot/pip/pyramid-googleauth-1.0.4` 
- One of the functional tests will fail
```
.tox/functests/bin/pytest tests/functional/checkmate/views/ui/admin_test
```

- Install this version of pyramid-googleauth
```
.tox/functests/bin/pip install -e /home/marcos/hypo/pyramid-googleauth
```

- Run the test again and it will pass this time.

### CODE REVIEW

**Review by seanh — CHANGES_REQUESTED** [review]
`pip` tip: you can `pip install` directly from a branch of a GitHub repo, this way the user doesn't need to check out the branch in a local copy of `pyramid-googleauth` just to install it in Checkmate: `.tox/functests/bin/pip install git+https://github.com/hypothesis/pyramid-googleauth.git@include-templates` (but this is not an editable install).

The fix works for me locally :+1:

Good job we have functests as unit tests would not catch template problems.

This will collide with the cookiecutter because `setup.cfg` is a templated file:

```terminal
$ make template
$ git diff
diff --git a/setup.cfg b/setup.cfg
index 0505a48..c2f8f77 100644
--- a/setup.cfg
+++ b/setup.cfg
@@ -25,10 +25,6 @@ install_requires =
 [options.packages.find]
 where = src

-[options.package_data]
-pyramid_googleauth =
-    templates/*
-
 [options.entry_points]

 [pycodestyle]
```

The fix is to move the new stuff into `.cookiecutter/includes/setup.cfg` and then re-run `make template`:

```diff
diff --git a/.cookiecutter/includes/setup.cfg b/.cookiecutter/includes/setup.cfg
new file mode 100644
index 0000000..fb4b9e5
--- /dev/null
+++ b/.cookiecutter/includes/setup.cfg
@@ -0,0 +1,4 @@
+
+[options.package_data]
+pyramid_googleauth =
+    templates/*
diff --git a/setup.cfg b/setup.cfg
index 0505a48..0867440 100644
--- a/setup.cfg
+++ b/setup.cfg
@@ -25,10 +25,6 @@ install_requires =
 [options.packages.find]
 where = src
 
-[options.package_data]
-pyramid_googleauth =
-    templates/*
-
 [options.entry_points]
 
 [pycodestyle]
@@ -46,3 +42,7 @@ ignore =
 
     # Bare except. PyLint finds these for us so we don't need pycodestyle to.
     E722,
+
+[options.package_data]
+pyramid_googleauth =
+    templates/*
```

**Comment by marcospri** [comment]
Thanks @seanh, just adapted this to use the templating :+1:

File: setup.cfg [hunk 1]
```diff
@@ -42,3 +42,6 @@ ignore =
 
     # Bare except. PyLint finds these for us so we don't need pycodestyle to.
     E722,
+[options.package_data]
```
Comment (seanh) [inline setup.cfg:46]: If you want a blank line between the `[pycodestyle]` section in `setup.cfg` (`E722`) and this then `.cookiecutter/includes/setup.cfg` needs to start with a blank line.

(This is true of many of the includes files. I couldn't figure out a better way to implement it. For example I couldn't get it to work exactly right by using Jinja2's "whitespace control" feature. When an include file does exist you'll often want a blank line before its contents but when the same include file does not exist then you don't want to render an extra blank line.)

**Review by seanh — DISMISSED** [review]

**Comment by seanh** [comment]
`make template` runs cleanly for me :+1:

**Pull request merged**